stories/src/primatives/quote-item.jsx

```diff
@@ -36,6 +36,9 @@ color: ${colors.black};
   color: ${colors.black};
   text-decoration: none;
 }
+&:focus {
+  outline: 2px solid rebeccapurple;
```

Comment:
love it! However, can you please add a `purple` entry to the `constants > colors` export and use that in all the places?